lumera/assetlist.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "lumera",
+  "assets": [
+    {
+      "description": ".",
+      "extended_description": ".",
+      "denom_units": [
+        {
+          "denom": "ulume",
+          "exponent": 0,
+          "aliases": [
+            "microlume"
+          ]
+        },
+        {
+          "denom": "LUME",
+          "exponent": 6,
+          "aliases": [
+            "lume"
+          ]
+        }
+      ],
+      "base": "ulume",
+      "display": "LUME",
+      "name": "lume",
+      "symbol": "LUME",
+      "coingecko_id": "-",
+      "type_asset": "sdk.coin",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lumera/images/lumera.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lumera/images/lumera.png"
+        }
+      ],
+      "socials": {
+        "discord": "https://discord.com/invite/lumeraprotocol",
+        "github": "https://github.com/LumeraProtocol/lumera",
+        "telegram": "https://t.me/lumeraprotocol",
+        "twitter": "https://x.com/lumeraprotocol",
```

Comment:
this account doesn't exist

Reply:
The accounts wil new names will go live on/after May 1st